apps/fz_http/test/fz_http_web/live/notifications_live/index_test.exs

```diff
@@ -0,0 +1,65 @@
+defmodule FzHttpWeb.NotificationsLive.IndexTest do
+  @moduledoc """
+  Test adding and removing notifications from the notifications table.
+  """
+  use FzHttpWeb.ConnCase, async: true
+  alias FzHttp.Notifications
+
+  setup do
+    on_exit(fn -> Notifications.clear() end)
+  end
+
+  setup [:create_notification, :create_notifications]
+
+  test "add notification to the table", %{admin_conn: conn, notification: notification} do
+    path = Routes.notifications_index_path(conn, :index)
+
+    Notifications.add(notification)
+
+    {:ok, _view, html} = live(conn, path)
+
+    assert html =~ notification.user
+  end
+
+  test "clear notification from the table", %{admin_conn: conn, notification: notification} do
+    path = Routes.notifications_index_path(conn, :index)
+    {:ok, view, _html} = live(conn, path)
+
+    Notifications.add(notification)
+
+    view
+    |> element(".delete")
+    |> render_click()
+
+    html =
+      view
+      |> render()
+
+    refute html =~ notification.user
+  end
```

Comment:
```suggestion
  html =
    view
    |> element(".delete")
    |> render_click()
```

This actually returns html.

Reply:
This is a weird one. I re-rendered the html, because in the html returned by render_click the notification is still in the markup. It's kind of a hack, but I'm not sure what else I could do 🤷

Reply:
Well not so weird, because we're message passing to get the view to update (genserver -> pubsub -> live view) and I assume that's not sync.

Reply:
Gotcha! 👍